wcfsetup/setup/db/install.sql

```diff
@@ -1952,6 +1962,8 @@ ALTER TABLE wcf1_edit_history_entry ADD FOREIGN KEY (obsoletedByUserID) REFERENC
 
 ALTER TABLE wcf1_event_listener ADD FOREIGN KEY (packageID) REFERENCES wcf1_package (packageID) ON DELETE CASCADE;
 
+ALTER TABLE wcf1_flood_control ADD FOREIGN KEY (objectTypeID) REFERENCES wcf1_object_type (objectTypeID) ON DELETE CASCADE;
```

Comment:
The more I look at this key the less I like it:

1. It can easily create an deletion storm on package uninstallation.
2. It makes the `INSERT`s slower, because it needs to check `wcf1_object_type`.

As entries in `wcf1_flood_control` expiry naturally we probably can leave this FK out.